Data/Sys/GameSettings/GP6P01.ini

```diff
@@ -0,0 +1,154 @@
+# GP6P01 - Mario Party 6
+
+[Core]
+# Values set here will override the main Dolphin settings.
+
+[EmuState]
+# The Emulation State. 1 is worst, 5 is best, 0 is not set.
+EmulationStateId = 5
+EmulationIssues = Mini games tour bus needs emulate format changes to preview the mini games.
```

Comment:
Please don't include `EmulationStateId` and `EmulationIssues` in six-character INIs unless there actually are emulation issues that are specific to a certain release of a game. We typically don't include sections like `[Core]` and `[EmuState]` at all in six-character INIs. Here's an example of how they usually are formatted: https://github.com/dolphin-emu/dolphin/blob/c2594695b8beda137c808e666b0651a1f09a293c/Data/Sys/GameSettings/GOYE69.ini